src/Halogen/HTML/Core.purs

```diff
@@ -70,17 +70,18 @@ text :: forall p i. String -> HTML p i
 text = HTML <<< VDom.Text
 
 -- | A smart constructor for HTML elements.
-element :: forall p i. VDom.ElemName -> Array (Prop i) -> Array (HTML p i) -> HTML p i
-element = coe (\name props children -> VDom.Elem (VDom.ElemSpec Nothing name props) children)
+element :: forall p i. Maybe VDom.Namespace -> VDom.ElemName -> Array (Prop i) -> Array (HTML p i) -> HTML p i
+element ns =
+  coe (\name props children -> VDom.Elem (VDom.ElemSpec ns name props) children)
   where
```

Comment:
@garyb Do we consider these things part of the public API? This adds an argument to the function.